src/processors/models/processorsManager.js

```diff
@@ -40,6 +42,12 @@ module.exports.getProcessor = async function (processorId) {
 };
 
 module.exports.deleteProcessor = async function (processorId) {
+    const areTestsUsingTheProcessor = await testsManager.isTestUsingProcessor(processorId);
```

Comment:
i would add to the error all test names, maybe this means we can add a column to the view (name)

Reply:
done